src/timedate/timedated.c

```diff
@@ -950,6 +963,10 @@ static int method_set_ntp(sd_bus_message *m, void *userdata, sd_bus_error *error
                         return r;
         }
 
+        r = set_ensure_consume(&c->set_ntp_calls, &bus_message_hash_ops, sd_bus_message_ref(m));
+        if (r < 0)
+                return r;
```

Comment:
Hm, I noticed that if something below failed the message still ref()ed. Maybe, better to move the end of this function?